php/blocks/class-loader.php

```diff
@@ -203,6 +207,22 @@ public function render_block_template( $block, $attributes ) {
 			$type = array( 'preview', 'block' );
 		}
 
+		if ( 'edit' !== $context ) {
```

Comment:
This is fine as is, but maybe it'd be simpler to have this as an `else` block of the conditional above:

```php
if ( 'edit' === $context ) {
	$type = array( 'preview', 'block' );
} else {
```